### PR TITLE
Control layout

### PR DESCRIPTION
**Pull request by frodrigo**
Generalization of concept deployed in #25

### CODE REVIEW

**Comment by frodrigo** [comment]
I fixed the merge conflict.

File: src/border.ts [hunk 1]
```diff
@@ -32,31 +33,46 @@ const countryCodes: { [key in Country]: number } = {
...
 
-  return {
-    resetBorder() {
-      resetBorder(map);
-    },
-    setBorder(country: Country) {
-      setBorder(map, country);
-    },
-  };
+  _initialUpdate() {
```
Comment (GuillaumeAmat) [inline src/border.ts:52]: You can use `private` and `protected` in TypeScript ;)

**Review by GuillaumeAmat — CHANGES_REQUESTED** [review]

**Comment by GuillaumeAmat** [comment]
@frodrigo `index-initial-settings.html` does not display the correct setting :/ 

Capture d’écran 2021-04-02 à 22 47 28

No errors in the console though.

File: src/border.ts (hunk 1 above)
Comment (frodrigo) [reply src/border.ts:52]: Fixed

**Comment by frodrigo** [comment]
> @frodrigo `index-initial-settings.html` does not display the correct setting :/
> 
> Capture d’écran 2021-04-02 à 22 47 28
> 
> No errors in the console though.

Fixed. Refactoring issue.

File: src/border.ts (hunk 1 above)
Comment (GuillaumeAmat) [reply src/border.ts]: 👌

**Review by GuillaumeAmat — APPROVED** [review]

**Pull request merged**